pkg/testutil/testutil_windows.go

```diff
@@ -16,6 +16,12 @@
 
 package testutil
 
+import (
+	"golang.org/x/sys/windows/svc/mgr"
+	"sync"
+	"testing"
+)
```

Comment:
```go
import (
	"sync"
	"testing"

	"golang.org/x/sys/windows/svc/mgr"
)
```

Reply:
Will do! If I'm not on a  windows box unfortunately my editor doesn't automatically do this 😩